validate/resources/integrationnetQuickNodeSetup.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Integrationnet Node Quick Install
+slug: integrationnet-node-quick-install
+hide_table_of_contents: false
+---
+
+
+# Setting Up Your Integrationnet Node
+
+This guide is specifically for setting up your Integrationnet Node, and is opinionated with sensible defaults.
+
+## 1. Choose VPS Provider
+Refer to the [Constellation Network Setup Guides](https://docs.constellationnetwork.io/validate/setup-guides/) for choosing a VPS provider.
+
+## 2. Provision Your VPS
```

Comment:
Somewhere in this section or after we'd need information about which ports to open up on the VPS since they're non-standard and the node won't be able to connect without them.

Reply:
@AlexBrandes the guide does not setup any firewall rules on the machine. TCP 9000-9001 and 9010-9011 are needed right? But wouldn't that just work without additional configuration/ufw setup?
Though nice to have, is a firewall really needed?

I can also refer to setting this up on VPS level: https://docs.constellationnetwork.io/validate/setup-guides/do/sgDroplet, https://docs.constellationnetwork.io/validate/setup-guides/aws/sg and https://docs.constellationnetwork.io/validate/setup-guides/gcp/sg

Reply:
Many platforms block all traffic by default (AWS, for example) and so you need to open up ports to allow access. Other platforms have different rules (Digital Ocean is wide open, for example). Since this section isn't aimed at a specific hosting platform then it's important to indicate which ports need to be opened (inbound 9000-9001 and 9010-9011, outbound all).

It's also a really, really bad idea to leave all ports on your server open, especially when you have a hot key with 250k DAG accessible to it stored on the server. We can't recommend that to node operators at all.

Reply:
@AlexBrandes @marcinwadon I have added the part for the firewall. 
As this was specifically written for an Integration Net node, there won't be a hot wallet with DAG present at first. But that might change when promoted do main net if I understand correctly. Better to already include I agree.

@netmet1 I have used the full URLs now (also for pages in this repo), not sure if a router module should be used to refer to internal links instead.

Let me know if anything remains